ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/SandboxLedgerBackend.scala

```diff
@@ -36,7 +36,9 @@ class SandboxLedgerBackend(ledger: Ledger)(implicit mat: Materializer) extends L
       ledger
         .lookupContract(contractId)
         .map(_.collect {
-          case ac if Ref.Party fromString submitter exists ac.disclosedTo => ac.contract
+          case ac
+              if Ref.Party fromString submitter exists (p => ac.witnesses(p) || ac.divulgences(p)) =>
```

Comment:
please write down why you're doing this, linking to the issue.

Reply:
c745ed9c7b